### ISSUE TITLE
[Bug]: False positive PC0095 for Handler functions

### ISSUE THREAD

**Issue by guidorobben** [ALCops/Analyzers#276]
### Analyzer

PlatformCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

LC0095

### Description

The Handlers in test codeunit needs special signature. You just cannot delete the parameter. If I dont want to check the string, but just confirm it. 

Maybe turn off PC0095 for handler functions?

<img width="755" height="307" alt="Image" src="https://github.com/user-attachments/assets/0a91d9a4-22ef-4462-8a7a-4dab6bd12bea" />

### AL code to reproduce

```
    [MessageHandler]
    procedure MessageHandler(MessageText: Text[1024])
    begin
    end;

    [ConfirmHandler]
    procedure ConfirmHandler(Question: Text[1024]; var Reply: Boolean)
    begin
        Reply := true;
    end;
```

**Comment by Arthurvdv** [comment]
Thanks, I've resolved this in the release of `v0.8.2` (and included all the handler functions).

I'll close the issue here, but feel free to reopen if it's not working as expected.
